cogs/reddit.py

```diff
@@ -8,6 +8,8 @@
 import urllib
 from discord.ext import commands
 from utils.configManager import BotConfig, RedditConfig
+from utils.log import log
+from utils.redditHelpers import get_subpost, credit_embed

```

Comment:
I don't think putting `redditHelpers` in `utils` makes sense since it's only got to do with the reddit cog. `utils` is more for bot-wide utilities. Can you do this instead?
- Create a `reddit` module inside `cogs` and move `reddit.py` into `cogs/reddit`.
- Move this file to `cogs/reddit` and rename to `helper.py`.

The bit in `wingbot.py` where it finds and adds cogs _might_ need some tweaking though.

Reply:
I'll just define them outside the class but in the same file